letsencrypt-auto-source/rebuild_dependencies.py

```diff
@@ -42,13 +42,13 @@
     # Too touchy to move to a new version. And will be removed soon
     # in favor of pure python parser for Apache.
     'python-augeas': '0.5.0',
-    # Package enum34 needs to be explicitly limited to Python2.x, in order to avoid
-    # certbot-auto failures on Python 3.6+ which enum34 doesn't support. See #5456.
-    # TODO: hashin seems to overwrite environment markers in dependencies. This needs to be fixed.
-    'enum34': '1.1.6 ; python_version < \'3.4\'',
+    # Cryptography 2.9+ drops support for OpenSSL 1.0.1, but we still need to support it
```

Comment:
nit:
```suggestion
    # Cryptography 2.9+ drops support for OpenSSL 1.0.1, but we still want to support it
```
I think as soon as we have a reason to upgrade it, we should just do so and move on.